src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.html

```diff
@@ -1,12 +1,47 @@
 <sofie-row class="mb-3">
   <form [formGroup]="actionTriggerForm" class="left-container">
     <div [formGroupName]="'data'">
-      <sofie-text-input i18n-label label="global.label" [formControlName]="'label'"></sofie-text-input>
-      <sofie-keyboard-input [isRequired]="true" i18n-label label="action-triggers.shortcut.label" [formControlName]="'keys'" i18n-errorMessage errorMessage="action-triggers.shortcut.error">
+      <sofie-text-input
+        i18n-label
+        label="global.label"
+        [formControlName]="'label'">
+      </sofie-text-input>
+      <sofie-keyboard-input
+        [isRequired]="true"
+        i18n-label
+        label="action-triggers.shortcut.label"
+        [formControlName]="'keys'"
+        i18n-errorMessage
+        errorMessage="action-triggers.shortcut.error">
       </sofie-keyboard-input>
-      <sofie-keyboard-input i18n-label label="action-triggers.physical-mapping.label" [formControlName]="'mappedToKeys'" i18n-helpText helpText="action-triggers.physical-mapping-help.tooltip">
+      <sofie-button-group>
+        <sofie-button #toggleButton (click)="toggleKeyboardVisibility()" i18n>
+          edit-keyboard-mapping.toggle-virtual-keyboard.button
+        </sofie-button>
+        <sofie-spacer></sofie-spacer>
+      </sofie-button-group>
+      <div class="wrapper">
+        <div *ngIf="isKeyboardVisible" class="floating-keyboard">
```

Comment:
You don't need the `wrapper` the div.
It creates a large "gap" in the UI that makes the UI look less structed because there is a random gap in the middle of it.

Reply:
Wrapper nuked in latest push